adhocracy4/modules/templatetags/module_tags.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.assignment_tag
+def itemHasFeature(item, feature):
```

Comment:
I am wondering whether we want to have this as an assignment tag. Real world usage would probably look like this:

    {% item_has_feature idea 'crud' as idea_has_crud %}
    {% if idea_has_crud %}
    <a href="/edit">Edit idea</a>
    {% endif %}

Alternatives would be a filter:

    {% if idea | has_feature:"crud" %}
    <a href="/edit">Edit idea</a>
    {% endif %}

Or a custom tag (definitely not lazy mans choice):
  
    {% has_feature idea 'crud'  %}
    <a href="/edit">Edit idea</a>
    {% end_has_feature %}